src/sass/modules/_m-schemaform.scss

```diff
@@ -242,22 +242,23 @@ fieldset.schemaform-field-template {
 }
 
 //duplicated from claims-status.scss
-.ask-va-questions {
-  .talk {
+.talk {
+  margin: 0.5em 0;
+  padding: 0;
+}
+
+.phone-number {
+  margin: 0.5em 0 2em;
+}
+
+.phone-number-link {
+    font-weight: bold;
+    text-decoration: none;
     margin: 0.5em 0;
-    padding: 0;
-  }
-  .phone-number {
-    margin: 0.5em 0 2em;
-    a {
-      font-weight: bold;
-      text-decoration: none;
-      margin: 0.5em 0;
-    }
-  }
 }
```

Comment:
Since you modified the class names in `AskVAQuestions`, are the above rules still applied to the right elements? Should they just be nested under `.help` as `&-talk`, `&-phone-number`, etc.?

Reply:
Thanks @bshyong, we're tidying up claims styles, including removing nesting where possible, so I'll add the new `help-` prefix here as well, nice catch, and thanks for the reminder.